lib/message_customize/locale.rb

```diff
@@ -1,6 +1,7 @@
 module MessageCustomize
   module Locale
     @available_messages = {}
+    CHANGE_LOARD_ORDER_LOCALES_FILE_PATH = 'config/initializers/35_change_load_order_locales.rb'
```

Comment:
Is CHANGE_LOARD_ORDER_LOCALES_FILE_PATH a typo of CHANGE_LOAD_ORDER_LOCALES_FILE_PATH?

Reply:
This is typo 😭 
I will fix it now!

Reply:
Fixed: https://github.com/ishikawa999/redmine_message_customize/pull/34/commits/613704739801b98be9c1b5597669b455bb1c5868